### PR TITLE
IstioCon China 2023 Blog Post

### PR DESCRIPTION
**Pull request by kfaseela**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/blog/2023/istiocon-china/index.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+IstioCon is committed to the industry-leading service mesh that provides a platform to explore insights gained from real-world Istio deployments, engage in interactive hands-on activities, and connect with maintainers across the entire Istio ecosystem.
+
+Alongside our virtual IstioCon 2023 event, [IstioCon China 2023](https://www.lfasiallc.com/kubecon-cloudnativecon-open-source-summit-china/co-located-events/istiocon-cn/) was held on September 26 in Shanghai, China. Part of the KubeCon + CloudNativeCon + Open Source Summit China, the event was arranged and hosted by the Istio maintainers and the CNCF. We were very proud to have a strong program for IstioCon in Shanghai and pleased to bring together members of the Chinese Istio community. The event was a testament to Istio’s immense popularity in the Asia-Pacific ecosystem.
+
+{{< image link="./group-pic.jpg"
+    caption="IstioCon China 2023"
+    >}}
+
+IstioCon China  kicked off with an opening keynote from Program Committee members Jimmy Song and Zhonghu Xu. The event was packed with great content, ranging from new features to end user talks, with major focus on the new Istio Ambient Mesh.
```
Comment (ericvn) [inline content/en/blog/2023/istiocon-china/index.md:19]:
```suggestion
IstioCon China kicked off with an opening keynote from Program Committee members Jimmy Song and Zhonghu Xu. The event was packed with great content, ranging from new features to end user talks, with major focus on the new Istio Ambient Mesh.
```

File: content/en/blog/2023/istiocon-china/index.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+## On the show floor
+
+Istio had a full time kiosk in the project pavilion at KubeCon + CloudNativeCon + Open Source Summit China 2023 , with the majority of questions asked around Ambient Mesh. Many of our members and maintainers offered support at the booth, where a lot of interesting discussions happened.
+
+{{< image width="75%"
+    link="./istio-support-at-the-booth.jpg"
+    caption="KubeCon + CloudNativeCon + Open Source Summit China 2023, Istio Kiosk"
+    >}}
+
+Another highlight was the Istio steering members and authors of the Istio books "Cloud Native Service Mesh Istio" and "Istio: the Definitive Guide", Zhonghu Xu and Chaomeng Zhang, spent time at the Istio booth interacting with our users and contributors.
```
[the remark this answers is not in the document]
Comment (dhawton) [reply content/en/blog/2023/istiocon-china/index.md:63]: Should Steering Committee be capitalized here as well? We capitalize it in the community repo https://github.com/istio/community/tree/master/steering

File: content/en/blog/2023/istiocon-china/index.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
+---
+title: "IstioCon China 2023 wrap-up"
+description: A quick recap of Istio at KubeCon + CloudNativeCon + Open Source Summit China, in Shanghai.
+publishdate: 2023-09-29
+attribution: "IstioCon China 2023 Programme Committee"
```
Comment (ericvn) [inline content/en/blog/2023/istiocon-china/index.md:5]: You have used both programme and program for the committee in this doc. I'm not sure what the official title is?

**Review by irisdingbj — APPROVED** [review]
LGTM, Thanks for putting this together!

File: content/en/blog/2023/istiocon-china/index.md (hunk 3 above)
Comment (kfaseela) [reply content/en/blog/2023/istiocon-china/index.md]: Fixed it

**Review by ericvn — APPROVED** [review]

**Pull request merged**